### PR TITLE
[quagga]: Disable ipv4 over ipv6 and enable ipv6 over ipv4 peer group

### PR DESCRIPTION
**Pull request by sihuihan88**

### CODE REVIEW

**Comment by stcheng** [comment]
@nikos-li please check @sihuihan88 patch on the configuration.

File: dockers/docker-fpm-quagga/bgpd.conf.j2 [hunk 1]
```diff
@@ -67,6 +68,10 @@ router bgp {{ DEVICE_METADATA['localhost']['bgp_asn'] }}
 {% if minigraph_devices[inventory_hostname]['type'] == 'ToRRouter' %}
   neighbor {{ neighbor_addr }} allowas-in 1
 {% endif %}
+  address-family ipv4
+    neighbor {{ neighbor_addr }} activate
```
Comment (lguohan) [inline dockers/docker-fpm-quagga/bgpd.conf.j2]: for ipv4 neighbor, allowas-in 1 is in address-family block, for ipv6 neighbor it is outside, is this intentional?

**Comment by nikos-li** [comment]
Everyone should be providing output from their Unit Testing ;-)

**Comment by sihuihan88** [comment]
After the fix:

router bgp 65100
 bgp router-id 10.1.0.32
 bgp log-neighbor-changes
 no bgp default ipv4-unicast
 bgp graceful-restart
 bgp bestpath as-path multipath-relax
 network 10.1.0.32/32
 network 192.168.0.0/27
 neighbor BGPSLBPassive peer-group
 neighbor BGPSLBPassive remote-as 65432
 neighbor BGPSLBPassive passive
 neighbor BGPSLBPassive ebgp-multihop 255
 neighbor BGPSLBPassive update-source Loopback0
 neighbor BGPSLBPassive activate
 neighbor BGPVac peer-group
 neighbor BGPVac remote-as 65432
 neighbor BGPVac passive
 neighbor BGPVac ebgp-multihop 255
 neighbor BGPVac update-source Loopback0
 neighbor BGPVac activate
 neighbor 10.0.0.57 remote-as 64600
 neighbor 10.0.0.57 description ARISTA01T1
 neighbor 10.0.0.57 activate
 neighbor 10.0.0.57 allowas-in 1
 neighbor 10.0.0.59 remote-as 64600
 neighbor 10.0.0.59 description ARISTA02T1
 neighbor 10.0.0.59 activate
 neighbor 10.0.0.59 allowas-in 1
 neighbor 10.0.0.61 remote-as 64600
 neighbor 10.0.0.61 description ARISTA03T1
 neighbor 10.0.0.61 activate
 neighbor 10.0.0.61 allowas-in 1
 neighbor 10.0.0.63 remote-as 64600
 neighbor 10.0.0.63 description ARISTA04T1
 neighbor 10.0.0.63 activate
 neighbor 10.0.0.63 allowas-in 1
 neighbor fc00::72 remote-as 64600
 neighbor fc00::72 description ARISTA01T1
 neighbor fc00::76 remote-as 64600
 neighbor fc00::76 description ARISTA02T1
 neighbor fc00::7a remote-as 64600
 neighbor fc00::7a description ARISTA03T1
 neighbor fc00::7e remote-as 64600
 neighbor fc00::7e description ARISTA04T1
 maximum-paths 64
 bgp listen range 10.154.239.128/26 peer-group BGPSLBPassive
 bgp listen range 192.168.0.0/27 peer-group BGPVac
!
 address-family ipv6
 network fc00:1::32/128
 neighbor BGPSLBPassive activate
 neighbor BGPVac activate
 neighbor fc00::72 activate
 neighbor fc00::72 allowas-in 1
 neighbor fc00::76 activate
 neighbor fc00::76 allowas-in 1
 neighbor fc00::7a activate
 neighbor fc00::7a allowas-in 1
 neighbor fc00::7e activate
 neighbor fc00::7e allowas-in 1
 maximum-paths 64
 exit-address-family
!

BGP table version is 0, local router ID is 10.1.0.32
Status codes: s suppressed, d damped, h history, * valid, > best, = multipath,
              i internal, r RIB-failure, S Stale, R Removed
Origin codes: i - IGP, e - EGP, ? - incomplete

   Network          Next Hop            Metric LocPrf Weight Path
*= 0.0.0.0          10.0.0.57                              0 64600 ?
*=                  10.0.0.61                              0 64600 ?
*=                  10.0.0.63                              0 64600 ?
*>                  10.0.0.59                              0 64600 ?
*> 10.1.0.32/32     0.0.0.0                  0         32768 i
*> 100.1.0.29/32    10.0.0.57                              0 64600 i
*> 100.1.0.30/32    10.0.0.59                              0 64600 i
*> 100.1.0.31/32    10.0.0.61                              0 64600 i
*> 100.1.0.32/32    10.0.0.63                              0 64600 i
*= 192.168.0.0/26   10.0.0.57                              0 64600 65100 ?
*=                  10.0.0.61                              0 64600 65100 ?
*=                  10.0.0.63                              0 64600 65100 ?
*>                  10.0.0.59                              0 64600 65100 ?

**Pull request merged**